packages/jupyter-ai/jupyter_ai/handlers.py

```diff
@@ -115,12 +115,21 @@ def get_chat_user(self) -> ChatUser:
         )
 
         if collaborative:
-            return ChatUser(**asdict(self.current_user))
+            names = self.current_user.name.split(" ", maxsplit=2)
```

Comment:
```suggestion
            names = self.current_user.name.split(" ", maxsplit=4)
```

For people using Spanish or French naming conventions, let's allow for up to 4 segments in a name.